internal/blockchain/ethereum/ffi_param_validator_test.go

```diff
@@ -108,7 +108,7 @@ func TestSchemaTypeInvalidFFIType(t *testing.T) {
 
 func TestSchemaTypeMissing(t *testing.T) {
 	_, err := NewTestSchema(`{}`)
-	assert.Regexp(t, "missing properties: 'type'", err)
```

Comment:
🙏 I'm guessing this is what has been intermittently failing my local test runs...